docs/examples.md

```diff
@@ -0,0 +1,184 @@
+# Examples
+
+Here you can find a bunch of simple examples for using fcct, with some explanations about what they do. The examples here are in no way comprehensive, for a full list of all the options present in ct check out the [configuration specification][spec].
+
+## Users and groups
+
+This example modifies the existing `core` user, giving it a known password hash (this will enable login via password), and setting its ssh key.
```

Comment:
We're disabling password auth by default (https://github.com/coreos/fedora-coreos-tracker/issues/138) so these won't actually work.